### PR TITLE
account-registration: Add examples of the 'working' cases.

### PR DESCRIPTION
**Pull request by progval**
Copy-pasted from what Ergo sends

### CODE REVIEW

**Comment by jwheare** [comment]
Nice, good to show one without verification too. And with optional email.

**Comment by progval** [comment]
I used a single example for both, so it does not get too long

File: extensions/account-registration.md [hunk 1]
```diff
@@ -218,6 +223,18 @@ The client can then proceed with the connection:
...
+
+    C: CAP LS 302
+    S: CAP * LS :draft/account-registration=before-connect
+    C: CAP REQ :draft/account-registration
+    C: CAP END
+    ...
+    C: REGISTER * * hunter2
+    S: REGISTER SUCCESS tester :Account successfully registered
+
+Note that email addresses may be required by the server, but not verified.
```
Comment (jwheare) [inline extensions/account-registration.md:236]: What does this mean? As in the server sends email-required but no VERIFICATION_REQUIRED?

Comment (progval) [reply extensions/account-registration.md:236]: Yes. Any suggestion to phrase it better?

Comment (jwheare) [reply extensions/account-registration.md:236]: I'd just leave it out, less confusing.

Comment (jwheare) [reply extensions/account-registration.md:236]: Better to just split the examples if you think people might be confused.

Comment (progval) [reply extensions/account-registration.md]: ok, done

**Pull request merged**